workers/unity/Packages/com.improbable.gdk.tools/LocalLaunch.cs

```diff
@@ -99,16 +108,33 @@ public static void Launch()
                 else
                 {
                     var content = File.ReadAllText(latestLogFile.FullName);
-                    message = $"{message}\n{content}";
-
-                    Debug.LogError(message);
+                    Debug.LogError($"{message}\n{content}");
                 }
 
                 process.Dispose();
                 process = null;
             };
         }
 
+        private static string GetCommand()
```

Comment:
these two methods can be inlined again.

Reply:
These definitely shouldn't be two functions; they both rely on specialised output of the other. This also needs a more descriptive name.

Reply:
Merged them now 👍 - still not sure if the names are descriptive though.